Notifier-service/src/services/conference.ts

```diff
@@ -3,30 +3,33 @@ import { ConferenceModel } from '../interfaces/models/conference'
 import { Conference } from '../schemas/conferences';
 import { Logger } from '../utility/log';
 import { injectable } from 'inversify';
+import { ConferenceModelMongo } from '../models/conference';
 
 @injectable()
 export class ConferenceServiceI extends ConferenceService {
     private logger = new Logger(this.constructor.name).getLogger()
-    constructor(private conferenceModel: ConferenceModel) {
+    constructor(private conferenceModel: ConferenceModel,
+                private conferenceModelMongo:ConferenceModelMongo) {
         super()
     }
-    async getOne(): Promise<Conference | null> {
+    async getOne(queryKey: string, queryValue: string): Promise<Conference | null> {
```

Comment:
* restrict it to using , only the _id. A query of this sort will lead to two things 
1. Someone working with the controller having a knowledge of db + what are valid fields for queries. (sepparate out concerns)
2. The same cannot be propagated to the routes also i.e eventually we will have to fix what information we will be using to query the data. So have it fixed through out the all ur code base.

Reply:
Sure